sql/mob_droplist.sql

```diff
@@ -26777,6 +26777,10 @@ INSERT INTO `mob_droplist` VALUES (3224,0,0,1000,2815,@COMMON);  -- Aptant Of Ar
 -- ZoneID: 125 - Picolaton
 INSERT INTO `mob_droplist` VALUES (3225,0,0,1000,11343,@COMMON); -- Thrakon Breastplate (Common, 15%)
 
+-- Garrison Drops
```

Comment:
Do mobs in garrison drop anything else apart from these? I don't remember if they do or not

Reply:
possibly seals/geodes but I don't even remember seeing those either